### PR TITLE
LPS-133634 Notify users on Revision Date

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
To be notified, you must be subscribed to the folder that contains the file.

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -2926,6 +2973,167 @@ private boolean _isValidFileVersionNumber(String version) {
...
+		User user = userLocalService.fetchUser(fileVersion.getUserId());
+
+		if (user == null) {
+			return;
+		}
+
+		DLGroupServiceSettings dlGroupServiceSettings =
+			DLGroupServiceSettings.getInstance(fileVersion.getGroupId());
+
+		if (emailType.equals("review") &&
```
Comment (adolfopa) [inline portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java:2994]: Why not
```
if (!emailType.equals(...) || !dlGroupServiceSettings.isEmailFileEntryReviewEnabled()) {
    return;
}
```

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1257401_1677

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1257401_1715

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1257401_1715

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
@adolfopa the error https://github.com/liferay-lima/liferay-portal/pull/1956#issuecomment-865367446 can have something related with the changes made on https://github.com/liferay-lima/liferay-portal/pull/1956/commits/45a7d52eea4d109e6e49d696a57258e2e5e06195 but I'm totally lost....

or it's unrelated?

can you give me a hint?

Thanks!

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by Hanlf** [comment]
PortalSmokeUpgrade tests should fail at this bug: [LPS-134610](https://issues.liferay.com/browse/LPS-134610) and the failures are not related to the commits.

**Comment by Hanlf** [comment]
Resent it to BChan here: https://github.com/brianchandotcom/liferay-portal/pull/103562

**Pull request closed without merging**